### PR TITLE
Fix migration doc for internal admin role rename

### PR DESCRIPTION
**Pull request by tharikaGitHub**
## Purpose
Fixes https://github.com/wso2/docs-apim/issues/6071

### Changes done

1. Removed the following point and the note.
Screenshot 2022-07-21 at 12 59 30

Screenshot 2022-07-21 at 12 59 52

2. Added the following note to the IS as KM migration docs only.
Screenshot 2022-07-26 at 11 43 48

3. Added the following to the IS as KM documentation

Screenshot 2022-07-26 at 11 44 37

4. Also added the following to the APIM migration document.

Screenshot 2022-07-24 at 09 45 29

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/install-and-setup/upgrading-wso2-api-manager/310-to-410/upgrading-from-310-to-410.md [hunk 1]
```diff
@@ -505,23 +505,36 @@ Follow the instruction below to upgrade the Identity component in WSO2 API Manag
 
 7. Copy the `org.wso2.carbon.is.migration-x.x.x.jar` from the `<IS_MIGRATION_TOOL_HOME>/dropins` directory to the `<API-M_4.1.0_HOME>/repository/components/dropins` directory.
 
-8. Add below configuration  to `<API-M_4.1.0_HOME>/repository/conf/deployment.toml` to disable group role separation.
+    !!! important
+            In WSO2 Identity Server 5.11.0, groups include user store roles and roles include internal roles. Therefore, from IS 5.11.0 onwards, we cannot have the same admin role in both primary and internal user domains. If the same admin role exists in both UM domains of your older version, you have to rename the internal admin role into a different role name. To do that, you have to follow the below steps.
```
Comment (binodmx) [inline en/docs/install-and-setup/upgrading-wso2-api-manager/310-to-410/upgrading-from-310-to-410.md:509]: Whether there are same admin roles in both primary and internal or not, user has to add following config

```
[super_admin]
create_admin_account = false 
```

shall we highlight this also?

Comment (tharikaGitHub) [reply en/docs/install-and-setup/upgrading-wso2-api-manager/310-to-410/upgrading-from-310-to-410.md:509]: Fixed with https://github.com/wso2/docs-apim/pull/6092/commits/4c3bfdcd44df9cb56c94cc61ba22c39c2e214938

File: en/docs/install-and-setup/upgrading-wso2-api-manager/300-to-410/upgrading-from-300-to-410.md [hunk 2]
```diff
@@ -506,11 +506,30 @@ Follow the instruction below to upgrade the Identity component in WSO2 API Manag
 
 7. Copy the `org.wso2.carbon.is.migration-x.x.x.jar` from the `<IS_MIGRATION_TOOL_HOME>/dropins` directory to the `<API-M_4.1.0_HOME>/repository/components/dropins` directory.
 
-8. Add below configuration  to `<API-M_4.1.0_HOME>/repository/conf/deployment.toml` to disable group role separation. 
-   
+    !!! important
+            In WSO2 Identity Server 5.11.0, groups include user store roles and roles include internal roles. Therefore, from IS 5.11.0 onwards, we cannot have the same admin role in both primary and internal user domains. If the same admin role exists in both UM domains of your older version, you have to rename the internal admin role into a different role name. To do that, you have to follow the below steps.
```
Comment (HiranyaKavishani) [inline en/docs/install-and-setup/upgrading-wso2-api-manager/300-to-410/upgrading-from-300-to-410.md:510]: AFAIK, with the fix given by IS, we didnt rename the internal admin, what it does is, they will add the primary/admin in to internal domain with given admin role in the toml configuration (admin_role = "<NEW-ADMIN-ROLE-NAME>") during the group role separation. @binodmx Am I correct or wrong?

**Review by samgnaniah — APPROVED** [review]

File: en/docs/install-and-setup/upgrading-wso2-api-manager/310-to-410/upgrading-from-310-to-410.md (hunk 1 above)
Comment (tharikaGitHub) [reply en/docs/install-and-setup/upgrading-wso2-api-manager/310-to-410/upgrading-from-310-to-410.md]: This configuration is not needed in the APIM only identity migration step. Therefore removed it.

File: en/docs/install-and-setup/upgrading-wso2-api-manager/300-to-410/upgrading-from-300-to-410.md (hunk 2 above)
Comment (tharikaGitHub) [reply en/docs/install-and-setup/upgrading-wso2-api-manager/300-to-410/upgrading-from-300-to-410.md]: Fixed with https://github.com/wso2/docs-apim/pull/6092/commits/ecd158325600f976aae1192b6b7ae465d34cd5a8

File: en/docs/install-and-setup/upgrading-wso2-api-manager/260-to-410/upgrading-from-260-to-410.md [hunk 3]
```diff
@@ -1144,45 +1144,23 @@ Follow the instruction below to upgrade the Identity component in WSO2 API Mana
...
-10. Update `<API-M_4.1.0_HOME>/repository/conf/deployment.toml` file as follows, to point to the previous user store.
+9. Update `<API-M_4.1.0_HOME>/repository/conf/deployment.toml` file as follows, to point to the previous user store.
 
     ```
     [user_store]
     type = "database"
     ```
 
-11. If you are migrating your user stores to the new user store managers with the unique ID capabilities, Follow the guidelines given in the [Migrating User Store Managers](https://is.docs.wso2.com/en/latest/setup/migrating-userstore-managers/) before moving to the next step
+10. If you are migrating your user stores to the new user store managers with the unique ID capabilities, Follow the guidelines given in the [Migrating User Store Managers](https://is.docs.wso2.com/en/latest/setup/migrating-userstore-managers/) before moving to the next step
```
Comment (samgnaniah) [inline en/docs/install-and-setup/upgrading-wso2-api-manager/260-to-410/upgrading-from-260-to-410.md:1161]:
```suggestion
10. If you are migrating your user stores to the new user store managers with the unique ID capabilities, follow the guidelines given in the [Migrating User Store Managers documentation](https://is.docs.wso2.com/en/latest/setup/migrating-userstore-managers/) before moving to the next step
```

File: en/docs/install-and-setup/upgrading-wso2-api-manager/300-to-410/upgrading-from-300-to-410.md [hunk 4]
```diff
@@ -486,7 +486,7 @@ Follow the instruction below to upgrade the Identity component in WSO2 API Manag
         order: 11   
     ```
 
-6. Remove the following 2 steps from  migration-config.yaml which is included under version: "5.11.0"
+6. Remove the following 3 steps from  migration-config.yaml which is included under version: "5.11.0"
```
Comment (samgnaniah) [inline en/docs/install-and-setup/upgrading-wso2-api-manager/300-to-410/upgrading-from-300-to-410.md:489]:
```suggestion
6. Remove the following 3 steps from `migration-config.yaml`, which is included under version: "5.11.0"
```

File: en/docs/install-and-setup/upgrading-wso2-api-manager/300-to-410/upgrading-from-300-to-410.md [hunk 5]
```diff
@@ -502,27 +502,23 @@ Follow the instruction below to upgrade the Identity component in WSO2 API Manag
...
-9. Update `<API-M_4.1.0_HOME>/repository/conf/deployment.toml` file as follows, to point to the previous user store.
+8. Update `<API-M_4.1.0_HOME>/repository/conf/deployment.toml` file as follows, to point to the previous user store.
 
     ```
     [user_store]
     type = "database"
     ```
 
-10. If you are migrating your user stores to the new user store managers with the unique ID capabilities, Follow the guidelines given in the [Migrating User Store Managers](https://is.docs.wso2.com/en/latest/setup/migrating-userstore-managers/) before moving to the next step
+9. If you are migrating your user stores to the new user store managers with the unique ID capabilities, Follow the guidelines given in the [Migrating User Store Managers](https://is.docs.wso2.com/en/latest/setup/migrating-userstore-managers/) before moving to the next step
```
Comment (samgnaniah) [inline en/docs/install-and-setup/upgrading-wso2-api-manager/300-to-410/upgrading-from-300-to-410.md:519]:
```suggestion
9. If you are migrating your user stores to the new user store managers with the unique ID capabilities, follow the guidelines given in the [Migrating User Store Managers documentation](https://is.docs.wso2.com/en/latest/setup/migrating-userstore-managers/) before moving to the next step
```

File: en/docs/install-and-setup/upgrading-wso2-api-manager/310-to-410/upgrading-from-310-to-410.md [hunk 6]
```diff
@@ -501,27 +501,23 @@ Follow the instruction below to upgrade the Identity component in WSO2 API Manag
...
-9. If the user store type in the previous version is set to "database" instead of default "database_unique_id", update `<API-M_4.1.0_HOME>/repository/conf/deployment.toml` file as follows, to point to the previous user store.
+8. If the user store type in the previous version is set to "database" instead of default "database_unique_id", update `<API-M_4.1.0_HOME>/repository/conf/deployment.toml` file as follows, to point to the previous user store.
 
     ```
     [user_store]
     type = "database"
     ```
    
-10. If you are migrating your user stores to the new user store managers with the unique ID capabilities, Follow the guidelines given in the [Migrating User Store Managers](https://is.docs.wso2.com/en/latest/setup/migrating-userstore-managers/) before moving to the next step
+9. If you are migrating your user stores to the new user store managers with the unique ID capabilities, Follow the guidelines given in the [Migrating User Store Managers](https://is.docs.wso2.com/en/latest/setup/migrating-userstore-managers/) before moving to the next step
```
Comment (samgnaniah) [inline en/docs/install-and-setup/upgrading-wso2-api-manager/310-to-410/upgrading-from-310-to-410.md:518]:
```suggestion
9. If you are migrating your user stores to the new user store managers with the unique ID capabilities, follow the guidelines given in the [Migrating User Store Managers documentation](https://is.docs.wso2.com/en/latest/setup/migrating-userstore-managers/) before moving to the next step.
```

File: en/docs/install-and-setup/upgrading-wso2-api-manager/320-to-410/upgrading-from-320-to-410.md [hunk 7]
```diff
@@ -263,27 +263,23 @@ Follow the instruction below to upgrade the Identity component in WSO2 API Mana
...
-9. If the user store type in the previous version is set to "database" instead of default "database_unique_id", update `<API-M_4.1.0_HOME>/repository/conf/deployment.toml` file as follows, to point to the previous user store.
+8. If the user store type in the previous version is set to "database" instead of default "database_unique_id", update `<API-M_4.1.0_HOME>/repository/conf/deployment.toml` file as follows, to point to the previous user store.
 
     ```
     [user_store]
     type = "database"
     ```
 
-10. If you are migrating your user stores to the new user store managers with the unique ID capabilities, Follow the guidelines given in the [Migrating User Store Managers](https://is.docs.wso2.com/en/latest/setup/migrating-userstore-managers/) before moving to the next step
+9. If you are migrating your user stores to the new user store managers with the unique ID capabilities, Follow the guidelines given in the [Migrating User Store Managers](https://is.docs.wso2.com/en/latest/setup/migrating-userstore-managers/) before moving to the next step
```
Comment (samgnaniah) [inline en/docs/install-and-setup/upgrading-wso2-api-manager/320-to-410/upgrading-from-320-to-410.md:280]:
```suggestion
9. If you are migrating your user stores to the new user store managers with the unique ID capabilities, follow the guidelines given in the [Migrating User Store Managers documentation](https://is.docs.wso2.com/en/latest/setup/migrating-userstore-managers/) before moving to the next step.
```

**Review by samgnaniah — APPROVED** [review]

**Pull request merged**